doc/classes/AudioStreamWAV.xml

```diff
@@ -11,6 +11,38 @@
 		<link title="Runtime file loading and saving">$DOCS_URL/tutorials/io/runtime_file_loading_and_saving.html</link>
 	</tutorials>
 	<methods>
+		<method name="load_from_buffer" qualifiers="static">
+			<return type="AudioStreamWAV" />
+			<param index="0" name="buffer" type="PackedByteArray" />
+			<param index="1" name="options" type="Dictionary" default="{}" />
+			<description>
+				Creates a new AudioStreamWAV instance from the given buffer. The keys and values of [param options] match the properties of [ResourceImporterWAV].
+				Usage of [param options] is identical to [method AudioStreamWAV.load_from_file].
+			</description>
+		</method>
+		<method name="load_from_file" qualifiers="static">
+			<return type="AudioStreamWAV" />
+			<param index="0" name="path" type="String" />
+			<param index="1" name="options" type="Dictionary" default="{}" />
+			<description>
+				Creates a new AudioStreamWAV instance from the given file path. The keys and values of [param options] match the properties of [ResourceImporterWAV]:
```

Comment:
```suggestion
				Creates a new [AudioStreamWAV] instance from the given file path. The keys and values of [param options] match the properties of [ResourceImporterWAV]:
```

Reply:
All seems good, might wanna go back at some point and do the same thing in `AudioStreamOggVorbis` but these changes make good sense.